library/oneview_datacenter.py

```diff
@@ -0,0 +1,209 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+###
+# Copyright (2016) Hewlett Packard Enterprise Development LP
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# You may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+###
+from ansible.module_utils.basic import *
+from hpOneView.oneview_client import OneViewClient
+from hpOneView.common import resource_compare
+
+DOCUMENTATION = '''
+---
+module: oneview_datacenter
+short_description: Manage OneView Data Center resources.
+description:
+    - "Provides an interface to manage Data Center resources. Can add, update, remove."
+requirements:
+    - "python >= 2.7.9"
+    - "hpOneView"
+author: "Gustavo Hennig (@GustavoHennig)"
+options:
+    config:
+        description:
+            - Path to a .json configuration file containing the OneView client configuration.
+        required: true
+    state:
+        description:
+            - Indicates the desired state for the Data Center resource.
+              'present' will ensure data properties are compliant to OneView.
+              'absent' will remove the resource from OneView, if it exists.
+        choices: ['present', 'absent']
+        required: true
+    data:
+        description:
+            - List with Data Center properties and its associated states.
+        required: true
+notes:
+    - "A sample configuration file for the config parameter can be found at:
+       https://github.com/HewlettPackard/oneview-ansible/blob/master/examples/oneview_config-rename.json"
+'''
+
+EXAMPLES = '''
+- name: Add a Data Center
+  oneview_datacenter:
+    config: "{{ config }}"
+    state: present
+    data:
+        name: "MyDatacenter"
+        width: 5000
+        depth: 6000
+        contents:
+            # You can choose either resourceName or resourceUri to inform the Rack
+            - resourceName: '{{ datacenter_content_rack_name }}' # option 1
+              resourceUri: ''                                    # option 2
+              x: 1000
+              y: 1000
+  delegate_to: localhost
+
+- name: Update the Data Center
```

Comment:
I think this example is unnecessary because it is almost equal to the previous one